local-helm/README.md

```diff
@@ -29,6 +29,10 @@ helm install bc stable/nginx-ingress
 
 Create the buying catalogue namespace - `kubectl apply -f local-namespace.yml`
```

Comment:
I didn't know I had to be in the directory of the platform repository locally within Powershell at this point. Some indication to tell the user this may lead to less confusion. Plus want to ensure the user is using the correct document which is in the local-helm folder.

Reply:
amended instructions to indicate desired directory